portfolio/src/main/webapp/script.js

```diff
@@ -14,7 +14,7 @@
 
 "use strict"
 
-// is called when page loads
+/* is called when page loads */
```

Comment:
Please avoid mixing functional and non-functional (style, etc) changes in one PR. Non-functional changes distract from the functional ones, and make the reviewing process slower. If you'd like to make style changes, you can create a separate PR, and clearly state that it only addresses style and, say, typos in comments - i.e. no functional changes. Such PR is easy to review, and hence fast to merge.

Reply:
Okay, thanks, will do!
In the next PR there also will be mixed style and functional changes, sorry for that, I just have already made it